src/clj/puppetlabs/services/jruby/jruby_puppet_schemas.clj

```diff
@@ -108,7 +115,8 @@
    :borrow-timeout schema/Int
    :max-active-instances schema/Int
    :max-requests-per-instance schema/Int
-   :use-legacy-auth-conf schema/Bool})
+   :use-legacy-auth-conf schema/Bool
+   (schema/optional-key :environment-class-cache-enabled) schema/Bool})
```

Comment:
Ah, yep, you're right.  Suppose I'd be in favor of dissoc'ing if it doesn't otherwise make sense to pass it around elsewhere in the "run-time" config that the jruby-puppet-pooled-service uses.  @cprice404, what do you think?